include/oneapi/tbb/detail/_config.h

```diff
@@ -545,4 +546,8 @@
 #define __TBB_PREVIEW_BLOCKED_ND_RANGE_DEDUCTION_GUIDES 1
 #endif
 
+#if !__TBB_DISABLE_SPEC_EXTENSIONS
+#define TBB_EXT_CUSTOM_ASSERTION_HANDLER 1
```

Comment:
Is the macro defined to a true/false value or a version value? If the former, should there be any value at all?
@vossmjp 

By the way, should we have a test that this macro is defined (to a certain value)?

Reply:
For a non-extension feature, the user can check ONETBB_SPEC_VERSION to determine which version is supported.  Since an EXT is not in the spec, seems like it would be good to have a version value in case it changes later in an incompatible way. We discussed using values like C++ feature test macros, so year and month. For this release, that would be 202510 to align with the upcoming release.

Reply:
> By the way, should we have a test that this macro is defined (to a certain value)?

Thanks, with a non-binary value it makes some sense.